tests/rptest/tests/compatibility/sarama_test.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import subprocess
+import time
+import requests
+from ducktape.mark.resource import cluster
+
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk import RpkTool
+
+#Compatibility testing for Sarama
+class SaramaTest(RedpandaTest):
+
+    CLIENT_DIR = "/kafka-clients"
+    TESTS_DIR = os.path.join(CLIENT_DIR, "sarama")
+
+    def __init__(self, test_context):
+        super(SaramaTest, self).__init__(test_context=test_context)
+        self._rpk = RpkTool(self.redpanda)
+
+        self.sarama_topic = 'sarama'
+
+        # Check if CLIENT Dir exists first
+        try:
+            os.mkdir(SaramaTest.CLIENT_DIR)
+        except FileExistsError:
+            self.logger.debug("Not creating {}. It already exists".format(SaramaTest.CLIENT_DIR))
+
+        # Check if sarama dir already exists first
+        if os.path.isdir(SaramaTest.TESTS_DIR):
+            self.logger.debug("Not downloading {}. It already exists".format(SaramaTest.TESTS_DIR))
+        else:
+            self._execute("git clone https://github.com/Shopify/sarama.git", SaramaTest.CLIENT_DIR)
```

Comment:
Right. This is a dup. Let’s add it as the image base layer

Reply:
Will do.